src/PrintService.java

```diff
@@ -0,0 +1,223 @@
+import faculty.*;
+
+public class PrintService {
+
+//Список студентов Гриффиндора
+    public void print(Gryffindor[] gryffindorStudents) {
+        System.out.println("\nНа Грффиндоре учатся:");
+        System.out.println("Имя         Фамилия         Сила магии      Расстояние трансгрессии     Благородство    Честь           Храбрость\n");
+        for (int i = 0; i < gryffindorStudents.length; i++) {
+            Gryffindor gryffindor = gryffindorStudents [i];
+            System.out.println(gryffindor.getName() +
+                    "       " + gryffindor.getSurname() +
+                    "             " + gryffindor.getSpellPower() +
+                    "               " + gryffindor.getTransgessionDistance() +
+                    "                       " + gryffindor.getGenerosity() +
+                    "               " + gryffindor.getHonor() +
+                    "               " + gryffindor.getBravery());
+        }
```

Comment:
дублирование кода - надо просто вызвать уже написанный метод
```suggestion
            currentGryffindorStudentPrint(i, gryffindorStudents);
        }
```